valusets/Valueset-England-ERSSpecialtyCategory.xml

```diff
@@ -0,0 +1,26 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="England-ERSSpecialtyCategory" />
+    <url value="https://fhir.nhs.uk/ValueSet/England-ERSSpecialtyCategory" />
+    <version value="1.0.0" />
+    <name value="EnglandERSSpecialtyCategory" />
+    <title value="England ERS Specialty Category" />
+    <status value="draft" />
+    <date value="2023-06-01" />
+    <publisher value="NHS England" />
+    <contact>
+        <name value="NHS England" />
+        <telecom>
+            <system value="email" />
+            <value value="interoperabilityteam@nhs.net" />
+            <use value="work" />
+            <rank value="1" />
+        </telecom>
+    </contact>
+    <description value="A valueset to describe the specialty of request. The values must be downloaded directly from ERS." />
```

Comment:
Speciality misspelt

Reply:
Specialty is the correct spelling in this domain